contrib/systemd/master/atomic-openshift-master-api

```diff
@@ -0,0 +1,64 @@
+#! /bin/bash
+
+set -euo pipefail
+
+pods_dir=/etc/origin/node/pods
+pods_disabled_dir=/etc/origin/node/pods.disabled
+
+static_pod_file=apiserver.yaml
+
+# Make sure both directories exist
+mkdir -p $pods_dir $pods_disabled_dir
+
+function get_container_values() {
```

Comment:
we are seeing in a cluster that it is possible to get more than one value for this filter if a container was not previously terminated.  we need to handle that case gracefully (probably choose the newest)

Reply:
While, I am not 100% sure how to reproduce it in the kube cluster, I tested the following scenario locally

- `docker run nginx` in 2 terminals creating two containers with an gap of 2 seconds.
- Ran `docker ps --latest` and it returned only one that was brought up later(or the newest one).

@derekwaynecarr - Is this the same thing you wanted?

Reply:
@ravisantoshgudimetla yes